python/ray/util/client/worker.py

```diff
@@ -281,7 +282,7 @@ def put(self, vals, *, client_ref_id: bytes = None):
             out = out[0]
         return out
 
-    def _put(self, val, *, client_ref_id: bytes = None):
+    def _put(self, val, client_ref_id: bytes):
```

Comment:
Any reason for dropping the `*` here?

Reply:
The `client_ref_id` arg is converted from optional keyword to become positional and always required. IIUC `*` becomes incorrect.